CodeEdit/Features/Editor/Views/CodeFileView.swift

```diff
@@ -109,7 +115,7 @@ struct CodeFileView: View {
 
     var body: some View {
         CodeEditSourceEditor(
-            $codeFile.content,
+            sourceEditorText,
```

Comment:
What do you think about this approach? Then we will don't need this `sourceEditorText` anymore

```swift
CodeEditSourceEditor(
       Binding(
            get: { codeFile.content ?? "" },
            set: { codeFile.content = $0 }
        )
}
```

Reply:
That will work. But using the _sourceEditorText_ variable can make the code more readable. 

I’m not advanced but I think creating the binding once in the initializer versus everytime in a computed property is a better way of doing things.